src/VolleyManagement.Backend/VolleyManagement.Services/TeamService.cs

```diff
@@ -105,6 +105,32 @@ public Team Create(CreateTeamDto teamToCreate)
             return _teamRepository.Add(teamToCreate);
         }
 
+        /// <summary>
+        /// Add players to the team.
+        /// </summary>
+        /// <param name="team">Team id to which players must be added.</param>
+        /// <param name="players">Player's ids that must be added.</param>
+        public void AddPlayers(TeamId team, IEnumerable<PlayerId> players)
+        {
+            _authService.CheckAccess(AuthOperations.Teams.Edit);
+            var changedTeam = Get(team.Id);
```

Comment:
what if team with such id does not exist?

Reply:
fixed